packages/runner/src/suite.ts

```diff
@@ -407,6 +407,20 @@ function createSuiteCollector(
     task,
     clear,
     on: addHook,
+    fixtures() {
+      return getCollectorFixture(suite)
```

Comment:
Does `getCollectorFixture/setCollectorFixture` need to go throug weakmap?
It looks like this is only used here, so why not a local var like `let collectorFixtures: FixtureItem[] | undefined`?